platform/common/export/common/gpio.hpp

```diff
@@ -0,0 +1,46 @@
+#ifndef COMMON_GPIO_HPP_
+#define COMMON_GPIO_HPP_
+
```

Comment:
it is really platform dependent. Maybe it is fine for now, but we need to think about better solution.